sql/template_xgboost.go

```diff
@@ -134,8 +134,11 @@ dpred = xgb_dataset('predict.txt', """{{.PredictionDatasetSQL}}""")
 bst = xgb.Booster({'nthread': 4})  # init model
 bst.load_model("{{.Save}}")  # load data
 preds = bst.predict(dpred)
-# TODO(typhoonzero): regression models may have different behavior
-pred_classes = np.argmax(np.array(preds), axis=1)
+
+# TODO(Yancey1989): using the train parameters to decide regressoin model or classifier model
+if len(preds.shape) == 2:
```

Comment:
Should find a more accurate way to determine whether the model is a regression model.